File: src/main/java/net/hectus/neobb/turn/person_game/categorization/Category.java
package net.hectus.neobb.turn.person_game.categorization;

import net.kyori.adventure.text.format.TextColor;
import org.bukkit.Material;

public interface Category {
    TextColor categoryColor();
    default int categoryMaxPerDeck() { return 1; }
    Material categoryItem();
    String categoryName();
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/person_game/categorization/Category.kt
package net.hectus.neobb.modes.turn.person_game.categorization

import net.kyori.adventure.text.format.TextColor
import org.bukkit.Material

interface Category {
    val categoryColor: TextColor
    val categoryMaxPerDeck: Int get() = 1

    val categoryItem: Material
    val categoryName: String
}
